about/our-team.php

```diff
@@ -2,64 +2,64 @@
 require('../template/top.php');
 head('Our Team', true);

```

Comment:
is this file worth updating?

Reply:
yes, somehow the versioned copy was totally different to what I had in production, but the prod version was in the git history but the changes were not newer, so idk, i just blew the old one away in the merge which fixed it